spark3/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -755,6 +756,27 @@ private static String sqlString(org.apache.iceberg.expressions.Literal<?> lit) {
     return null;
   }
 
+  /**
+   * Returns an Iceberg Table by its name from a Spark V2 Catalog. The {@link TableOperations} of the table may be
```

Comment:
Thanks for adding, can we add 'Depending on the type of Spark Catalot, the TableOperations of the table..."

So it's clear to the user why it would be stale.

Reply:
Sorry just saw the other comments.  If you guys think its ready its fine, to me  just the last sentence seemed a bit out of context

Reply:
Updated the Java doc.